scripts/generate_error_code.hs.license

```diff
@@ -0,0 +1,2 @@
+SPDX-FileCopyrightText: 2021 Tocqueville Group
```

Comment:
nitpick, but I think that this might be included in the actual file (after the shebang)

Reply:
Updated this.